distribution/packages/build.gradle

```diff
@@ -335,7 +335,6 @@ Closure commonDebConfig(String architecture) {
 
     // versions found on oldest supported distro, centos-6
     requires('bash', '4.1', GREATER | EQUAL)
-    requires('lsb-base', '4', GREATER | EQUAL)
```

Comment:
Library `lsb-base` is now abandoned and marked as obsolete by Lintian. This lib is not available in Debian 12.
As far as I understand it can be used for some logging function, but I couldn't find any direct calls to it from ES code.
The tests has passed on all DEB-based distributions so I assume it is safe to delete it.

Reply:
Presumable it's just because it brings in `sysvinit-utils`. @rjernst do you recall why this might be needed?